google-cloud-firestore/acceptance/firestore/aggregate_query_test.rb

```diff
@@ -81,13 +81,13 @@
       aq = @rand_query_col.aggregate_query
                           .add_count(aggregate_alias: 'one')
                           .add_count(aggregate_alias: 'one')
-      expect { snapshot = aq.get.first }.must_raise expected_error_class
+      expect { _ = aq.get.first }.must_raise expected_error_class
```

Comment:
Just omit the `_ =` altogether since we don't need to assign anything.

Reply:
ack